.travis.yml

```diff
@@ -0,0 +1,19 @@
+language: go
+
+go: stable
```

Comment:
should this be made specific to a version, to avoid build failures as go version changes.

Reply:
Can we use 1.9 or both 1.8 and 1.9 ?